Source/ZoomNet/Json/WebhookEventConverter.cs

```diff
@@ -418,6 +418,61 @@ public override Models.Webhooks.Event Read(ref Utf8JsonReader reader, Type typeT
 					meetingParticipantRoomSystemCalloutRingingEvent.Participant = ParseParticipantProperty<InvitedRoomParticipant>(payloadJsonProperty);
 					webHookEvent = meetingParticipantRoomSystemCalloutRingingEvent;
 					break;
+				case Models.Webhooks.EventType.MeetingAiCompanionAssetsDeleted:
+					var meetingAiCompanionAssetsDeleted = payloadJsonProperty.ToObject<MeetingAiCompanionAssetsDeletedEvent>(options);
+					meetingAiCompanionAssetsDeleted.DeletedAssets = payloadJsonProperty.GetProperty("object/ai_companion/deleted_assets", true).Value.ToObject<string[]>();
```

Comment:
This can be simplified:

```suggestion
					meetingAiCompanionAssetsDeleted.DeletedAssets = payloadJsonProperty.GetPropertyValue<string[]>("object/ai_companion/deleted_assets");

```

Reply:
Thanks, applied.